Updated Stuff and Added TopCommand

kimunachan opened the pull request.

kimunachan committed:
Added Top Command
with associated Event

kimunachan committed:
Added Top Command
with associated Event

kimunachan committed:
Updated Bot

FlayInAHook reviewed (changes requested):
announcementUserLevelUP() bitte überarbeiten. Genauer gesagt bitte eine Config für die level up Rollen machen

kimunachan committed:
Added LevelRoleConfig
Updated ConfigType
Implemented LevelroleConfig into announcementUserLevelUp function

FlayInAHook reviewed (changes requested):
bitte die anpassungen aus dc vornehmen :)

kimunachan committed:
Updated LevelRoleConfig and LevelingHandler
- Changed the way the Map works
- All Roles in one List
  - Removed Static Roles
Cleaned up Eclipse automated Todo crap

FlayInAHook reviewed (changes requested):
siehe discord

kimunachan committed:
Updated LevelRoleConfig and LevelingHandler
Added TopCommand and Handler back
Cleaned up System outs

kimunachan committed:
Fixed Level role assignment
